src/Lib/BSP/IO/GameLumpHeader.cs

```diff
@@ -0,0 +1,31 @@
+namespace Lumper.Lib.BSP.IO
+{
+    public class GameLumpHeader
+    {
+        public GameLumpHeader()
+        { }
+
+        public GameLumpHeader(LumpHeader tmpHeader, ushort version, int id)
+        {
+            Id = id;
+            Fileofs = (int)tmpHeader.Offset;
+            //set last bit to 0 (only seen flags as 1 and 0 so far but its probably a bitfield)
+            Flags &= 0xFFFE;
+            //length is always the uncompressed length
+            Filelen = (int)tmpHeader.UncompressedLength;
```

Comment:
var names

Reply:
I really think these matter here since they're properties. So `FileLen` and `FileOfs` (only capitalise 2-letter initials)